requirements/dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Does it make a difference that we are generating this with python 3.8 but the docker container is running python 3.10?